crates/context/src/lib.rs

```diff
@@ -239,11 +240,25 @@ impl ContextManager {
                 .send(*context_secret)
                 .await?;
 
+            let proxy_contract = this.get_proxy_contract(context.id).await?;
+            drop(this.add_context(
```

Comment:
why drop the result?

Reply:
also, as mentioned earlier, state initialization (which is above) requires `add_context` to have been called

so restore [this](https://github.com/calimero-network/core/pull/944/files#r1832510769)

and here, instead of `add_context`, fetch the context config params and just update the proxy contract.